### ISSUE TITLE
Make updates from IETF 106

### ISSUE THREAD

**Issue by jlivingood** [alex-nicat/ietf-dprive-phase2-requirements#21]
Jason will take 1st pass.

threat model -
- Attacker w/access to all traffic out of scope? EKR does not agree.

DoT always required?
- YES
- Maybe other equivalent secure transports (DoH, DoQ)
- Rephrase to require secure transport
- Per Ben also explain pros/cons of diff privacy prot practices at each layer of lookup

Trust anchor/authority: Should this depend only on the DNS, such as DANE, or also on Certification Authorities? 
- Choices: out of band, DANE/DNS, PKI/CA
- Key point is referential integrity (per EKR - good re-wording)
- Note circular dependency risks (list potential dependencies and say not to break these)
- Must be simple enough to be (externally) analyzable & diagnosable 
- Should we look at happy eyeballs to see if we can learn from that? 
- Would be nice a la happy eyeballs to retain some data for a period of time / subsequent queries

Downgrade Prevention & Preferences: Should the user (e.g. stub, app, resolver) be able to say only ever use DoT and never downgrade? Or should the authoritative domain be the only one? Or should both be permitted and let the app/stub decide what to do based on that info?
- Domain must specify 53 or DoT or both or only DoT if avail
- look at rfc7762
- fallback logic/requirements needed
- how to deal w/happy eyeballs for auth server
- disc & advert of non-downgradable protocols

Discovery: What requirements do we have for a recursive to determine availability of encryption on an authoritative server? (e.g. loose coupling vs. whitelists are okay)
- Has to be in dns, signed, linked to nameserver
- use dns

high level
- recommended/mandatory is important discussion to happen next
- solutions can happen then in parallel
- major missing - zones whose NSs are not all controlled by one org (eg CDNs)
- DANE from apex recommended over delegation glue

**Comment by jlivingood** [comment]
made these changes
